src/documents/modals/UploadWizard/UploadSteps/VersionSummaryStep.js

```diff
@@ -39,7 +39,7 @@ const VersionSummaryStep = ({fileToUpdate, file, setDescription}) => {
           changed in this version
         </h5>
         <Form>
-          <Form.Field required focus>
```

Comment:
📓 we need to make a PR in the future to make sure the `focus` and `tabindex` works correctly for forms